Formula/butteraugli.rb

```diff
@@ -0,0 +1,20 @@
+class Butteraugli < Formula
+  desc "Measuring differences between images."
+  homepage "https://github.com/google/butteraugli"
+  url "https://github.com/google/butteraugli/archive/2f8d6a0b1466b003d610271ef1f0ae3dd0d28998.tar.gz"
+  version "0.1"
+  sha256 "79ce7ead4ccb159b84bbab05e8a3e569c3b98b032ee58bdfd13df25acc85d5a2"
+
+  depends_on "libpng"
+
+  def install
+    cd "src"
+    system "make"
+    bin.install "compare_pngs"
+  end
+
+  test do
+    # unfortunatley, butteraugli fails when applied to the test fixtures
+    # system "#{bin}/compare_pngs", test_fixtures("test.png"), test_fixtures("test.png")
```

Comment:
I see. I think the best solution would be to switch the test to a short C++ program that exercises the library instead of this simple command line wrapper. It shouldn't be too difficult to generate some appropriately sized dummy images in memory. (As I'm understanding, the software is more about the library and the CLI is mostly of a demo use of that library.)

An alternative would be to generate a PNG of sufficient size and encode it directly in the `test do` block, write it to files in the test directory, and then use that. (A PNG stripped of all optional metadata and that compresses well should be doable in a few lines of Base64 encoding.)